.opencode/commands/career-ops-interview-prep.md

```
@@ -0,0 +1,12 @@
+---
+description: Full interview prep package for a specific company and role
+---
+
+Run full interview prep for the following company and role:
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)